### PR TITLE
added date and moved files to folders

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@Airdroplet @kalavathig1 I added the date to several files for the 2024 GA release so that the files show up on the website. I checked the versions in all files which did not have a date yet up to the versions included in the RC4.

File: content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ build_artifact:
   - value: tc-QHwMfWtBk7
     label: cumulocity
 ticket: MTM-41760
-version: 10.18.590.0
+version: 10.18.540.37
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md:15]: I checked the version in the ticket and I believe it must be changed.

File: content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md [hunk 2]
```diff
@@ -12,6 +12,6 @@ build_artifact:
   - value: tc-QHwMfWtBk7
     label: cumulocity
 ticket: MTM-55259
-version: 1018.504.17
+version: 1018.504.17 ??
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md:15]: @kalavathig1 Where does this version come from? There is no fix version for 2024 in the ticket. Should we remove this entry from the 2024 branch?

File: content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md (hunk 1 above)
Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md:15]: > @kalavathig1 I checked the version in the ticket and I believe it must be changed.

I just saw that we have a second PR open for this, actually with the same comment. I will close the other PR.

File: content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md (hunk 2 above)
Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md:15]: > @kalavathig1 Where does this version come from? There is no fix version for 2024 in the ticket. Should we remove this entry from the 2024 branch?

I just saw that we have a second PR open for this, actually with the same comment: https://github.com/SoftwareAG/c8y-docs/pull/1839. I will close this other PR and remove the file here.

File: content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md (hunk 1 above)
Comment (kalavathig1) [reply content/change-logs/platform-services/cumulocity-10-18-540-37-support-for-certificate-revocation-list.md:15]: yeah you right :)

**Review by kalavathig1 — APPROVED** [review]

**Pull request merged**